cmd/kubelet/app/options/options.go

```diff
@@ -416,7 +416,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 
 	fs.Int32Var(&c.RegistryPullQPS, "registry-qps", c.RegistryPullQPS, "If > 0, limit registry pull QPS to this value.  If 0, unlimited.")
```

Comment:
it doesn't seem like there's special handling for this one either, won't it be set to the default without special handling? I kind of feel like each of the QPS fields (these two and KubeAPI ) should all act consistently. I am not feeling super opinionated on how.